pkg/scheduler/apis/config/v1alpha1/defaults.go

```diff
@@ -111,4 +111,10 @@ func SetDefaults_KubeSchedulerConfiguration(obj *kubeschedulerconfigv1alpha1.Kub
 		defaultPodMaxBackoffSeconds := int64(10)
 		obj.PodMaxBackoffSeconds = &defaultPodMaxBackoffSeconds
 	}
+
+	// Enable profiling by default in the scheduler.
+	enableProfiling := true
```

Comment:
move it inside the if statement like the others.

Reply:
thanks ~ Done